airbyte-integrations/connectors/source-datascope/source_datascope/datascope.yaml

```diff
@@ -0,0 +1,68 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://www.mydatascope.com/api/external/"
+    http_method: "GET"
+    authenticator:
+      type: ApiKeyAuthenticator
+      header: "Authorization"
+      api_token: "{{ config['Authorization'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: DefaultPaginator
+      url_base: "*ref(definitions.requester.url_base)"
+      page_size_option:
+        inject_into: "request_parameter"
+        field_name: "limit"
+      pagination_strategy:
+        type: "OffsetIncrement"
+        page_size: 200
+      page_token_option:
+        inject_into: "request_parameter"
+        field_name: "offset"
+    requester:
+      $ref: "*ref(definitions.requester)"
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+  location_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "locations"
+      primary_key: "id"
+      path: "/locations"
+  answers_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "answers"
+      primary_key: "form_answer_id"
+      path: "/v2/answers"
```

Comment:
@jhammarstedt Datascope API allows start/end parameter which enables incremental syncs. This feature is very useful to users, what do you think about it? Let me know if you need any help to implement.

Reply:
It is the same for notifications endpoint.

Reply:
Hi, sure I can give incremental a go! I'll see if I can get it done over the weekend!